pkg/kp/kv.go

```diff
@@ -37,6 +38,18 @@ func NewStore(opts Options) *Store {
 	return &Store{client: client}
 }
 
+// KVError encapsulates an error in a Store operation. Errors returned from the
+// Consul API cannot be exposed because they may contain the URL of the request,
+// which includes an ACL token as a query parameter.
```

Comment:
Yeah you can't really figure out the contents of the error without reflection. Consul returns an `fmt.Errorf` for bad response codes so you'd actually have to parse a string to find it. I wish more go libraries were disciplined about the use of custom error values or types, but unfortunately this is not one of those cases. *glances nervously at our own `util.Errorf`*